modules/serverless-kn-container.adoc

```diff
@@ -1,13 +1,14 @@
 // Module included in the following assemblies:
 //
-// * serverless/develop/serverless-applications.adoc
 // * serverless/reference/kn-serving-ref.adoc
 
 :_content-type: REFERENCE
 [id="serverless-kn-container_{context}"]
 = Knative client multi-container support
 
-You can use the `kn container add` command to print YAML container spec to standard output. This command is useful for multi-container use cases because it can be used along with other standard `kn` flags to create definitions. The `kn container add` command accepts all container-related flags that are supported for use with the `kn service create` command. The `kn container add` command can also be chained by using UNIX pipes (`|`) to create multiple container definitions at once.
+You can use the `kn container add` command to print YAML container spec to standard output. This command is useful for multi-container use cases because it can be used along with other standard `kn` flags to create definitions.
```

Comment:
I did not see this section on the preview deployment! There is an instance of "instead of" that should be "rather than" on line 48. That's beyond this PR, though.

Reply:
Oops, sorry, it's because it used to be in this page but I moved it to just being here https://deploy-preview-44039--osdocs.netlify.app/openshift-enterprise/latest/serverless/reference/kn-serving-ref.html#serverless-kn-container_kn-serving-ref since it felt like it was too much additional info here where it wasn't required.